### PR TITLE
Populate

### PR DESCRIPTION
**Pull request by FranzWegener**
Added populate form functionality + Tests

### CODE REVIEW

File: src/Forms/Form.php [hunk 1]
```diff
@@ -74,19 +74,33 @@ public function renderJavascript()
     }
     /**
      * (non-PHPdoc)
-     * @see \QuantumForms\FormInterface::validateInput()
+     * @see \QuantumForms\FormInterface::validate()
      */
-    public function validateInput(array $request)
```
Comment (YupItsZac) [inline src/Forms/Form.php]: Why the change in name?

Comment (FranzWegener) [inline src/Forms/Form.php]: I thought the name is a bit long any cumbersome. I mean there's nothing else to validate other than input, right?

**Comment by YupItsZac** [comment]
+1 on the 3.5% increase! 👍 from me
